shm/src/print_cargo.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iomanip>
+#include <iostream>
+
+#include "player.hpp"
+#include "print_cargo.hpp"
+
+void PrintCargo::execute(Player* player) {
+    size_t cargoIndex = 0;
+    std::string dash(42, '-');
+    auto cargo = player->getCargo(cargoIndex);
+    
+    std::cout << std::setw(14) << "Name |" << std::setw(14) << " Amount |"
+              << std::setw(15) << " Baseprice |\n"
+              << dash << '\n';
+
+    while (cargo) {
+        std::cout << std::setw(12) << cargo->getName() << " |" << std::setw(12)
+                  << cargo->getAmount() << " |" << std::setw(12) << cargo->getBasePrice()
+                  << " |\n";
```

Comment:
I would put this into another f-ction called for example: displayCargoValues(cargo)